provider/blob/Cargo.toml

```diff
@@ -30,8 +30,8 @@ zerotrie = { workspace = true, features = ["serde", "zerovec"] }
 log = { workspace = true, optional = true }
 
 [dev-dependencies]
-icu_locid = { path = "../../components/locid", features = ["serde"] }
-icu_datagen = { path = "../../provider/datagen", default-features = false, features = ["networking"] }
+icu_locid = { workspace = true, features = ["serde"] }
```

Comment:
I think we cannot use local workspace dependencies here, as they have version numbers. We went back and forth on this before.